client/src/components/Tool/ToolCardSection.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+const props = defineProps({
+    title: String,
+});
+</script>
+
+<template>
+    <div class="mt-2 mb-4">
+        <div v-if="title" class="title-bar my-2">
+            <div class="stripe"></div>
+            <h2>{{ props.title }}</h2>
+            <div class="stripe"></div>
+        </div>
+
+        <div v-else class="stripe"></div>
```

Comment:
Please remove this line here. It produces the empty stripe/bar above the execute button, it feels unnecessarily separated by horizontal lines.